automate CSV translations

chriskjou opened the pull request:
Select any of the following which apply to your changes:

- [X] Bug fix
- [] New feature
- [X] Breaking change

Summarize your changes, including issues addressed:
- Added function that would wait for CSV parsing to remove copy/paste from Python file
- Added link to main.css to enable proper running of project
- Added JQuery and changed select dropdown function to ensure new dropdown selection would not evoke Javascript errors 

Describe how you tested your changes:
- Ran on local server host and checked if language/programming language were correctly changed when selected and the translations were updated and only words specific to each language/programming language were suggested in the translation
- Ensured that no Javascript errors were displayed on local host

### Checklist:

- [X] I have commented throughout my code
- [X] I have tested my code
- [X] I have modified the documentation, if necessary
- [X] I have modified information regarding dependencies, if necessary

fibanneacci commented:
Thank you so much!

chriskjou commented:
No problem! I saw your post on NCWIT and thought it was a cool project - I
had reached out on FB but not sure if you saw my message! Are there any
other ways I can help? (:

Hope you're having a great break!
ᐧ

On Thu, Dec 27, 2018 at 8:20 PM AL <notifications@github.com> wrote:

> Thank you so much!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fibanneacci/langplusplus/pull/10#issuecomment-450274174>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AU-A53bu5dH-_aVg-Ys-YTNgtAfZvBzSks5u9X_wgaJpZM4ZjQGE>
> .
>

fibanneacci commented:
Hi! I'm so sorry, I just noticed your facebook message :( the changes you made earlier were really helpful & I think addressed most of the issues with the functionality, the biggest thing missing now is probably translations if you're interested in working on those. Enhancements/new features are also welcome, at this point it's still very open-ended!

Thanks again, hope your break is going well too!

chriskjou commented:
No worries! I don't think I'm fluent enough in any other language to
translate for programming languages (lol), but if it involves scraping
other websites, etc. - I'm happy to help!

Thanks again - Happy New Year!
ᐧ

On Sun, Dec 30, 2018 at 7:46 PM AL <notifications@github.com> wrote:

> Hi! I'm so sorry, I just noticed your facebook message :( the changes you
> made earlier were really helpful & I think addressed most of the issues
> with the functionality, the biggest thing missing now is probably
> translations if you're interested in working on those. Enhancements/new
> features are also welcome, at this point it's still very open-ended!
>
> Thanks again, hope your break is going well too!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fibanneacci/langplusplus/pull/10#issuecomment-450600405>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AU-A5-iDQr1PpqmsHigWH23lC7j3nP6bks5u-Wx1gaJpZM4ZjQGE>
> .
>
